### PR TITLE
[Look&Feel] Consistency of Plus Icons

### PR DESCRIPTION
**Pull request by danieldong51**
### Description
Updated plus icon over plusInCircle for add/create use cases. Did not change instances of other use cases, or when minusInCircle was attached.

| Scope | Before (v7light) | After (v7light) | Before (v8dark) | After (v8dark) |
| ----- | ----- | ----- | ----- | ----- |
| Dashboards: Add Alert Monitor | Dashboards Add Monitor v7 Light Before | Dashboards Add Monitor v7 Light Post | Dashboards Add Monitor v8 Dark Before | Dashboards Add Monitor v8 Dark Post | 
| Add Action / Trigger | Add v7 Light Before | Add v7 Light Post | Add v8 Dark Before | Add v8 Dark Post |

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by virajsanghvi** [comment]
Could you actually revert this one - because it's an empty button, I think it looks weird without either a button container or the fill - i'd rather just leave it unchanged for now.
image

File: public/utils/contextMenu/actions.tsx [hunk 1]
```diff
@@ -81,7 +81,7 @@ export const getActions = () =>
       title: i18n.translate('dashboard.actions.alertingMenuItem.addAlertingMonitor.displayName', {
         defaultMessage: 'Add alerting monitor',
       }),
-      icon: 'plusInCircle' as EuiIconType,
+      icon: 'plus' as EuiIconType,
```
Comment (virajsanghvi) [inline public/utils/contextMenu/actions.tsx:84]: Do you have an screenshot of this one? Think its fine, but don't think it was in the summary

Comment (danieldong51) [reply public/utils/contextMenu/actions.tsx:84]: Edited with correct screenshots

**Review by virajsanghvi — APPROVED** [review]

**Comment by virajsanghvi** [comment]
Let's hold off on this one until we clarify Laura's guidance for Add's

**Review by bowenlan-amzn — APPROVED** [review]

**Comment by zhongnansu** [comment]
@danieldong51 is it ready for review? Could you converted PR from draft?

**Comment by danieldong51** [comment]
> @danieldong51 is it ready for review? Could you converted PR from draft?

Laura recommended me to not carry out these changes

**Review by amsiglan — APPROVED** [review]

**Pull request closed without merging**